tools/cldr-apps/src/test/java/org/unicode/cldr/util/SandboxLocalesTest.java

```diff
@@ -45,7 +46,14 @@ void test() throws IOException {
         // our factory includes common/main, so limit here.
         for(final CLDRLocale l : SpecialLocales.getByType(SpecialLocales.Type.scratch)) {
             System.out.println("Testing " + l);
-            CLDRFile f = factory.make(l.getBaseName(), true, null);
```

Comment:
Could you just omit the third argument? Then it uses the factory's default.  This was definitely a mistake, I didn't have a reason to pass null here.

Reply:
I will try that in a future cleanup, I would like to get this merged so we can start shakedown.

Reply:
I did that in https://github.com/unicode-org/cldr/pull/2009